deny.toml

```diff
@@ -65,14 +63,17 @@ allow = [
   "QPL-1.0",
   "RPSL-1.0",
   "SISSL",
-  "Sleepycat",
   "SPL-1.0",
-  "Unlicense",
+  "Sleepycat",
   "UPL-1.0",
+  "Unicode-3.0",
+  "Unlicense",
```

Comment:
What do we need these for? Might be worth providing the motivation in the commit log.

Reply:
done